tests/epyccel/test_kind.py

```diff
@@ -32,7 +32,7 @@ def real_greater_bool(x0, x1):
     assert(real_greater_bool(1.5,1.2)==epyc_real_greater_bool(1.5,1.2))
 
 def test_input_output_matching_types(language):
-    @types('float32', 'float32')
+    @types('real', 'real')
```

Comment:
Can you explain this change?

Reply:
- the argument feed to this function were python float.
- the precision of a python float is 64 bit on all system.

- now that we are collecting value as Pyobject I tried to avoid calling numpy and python check if the default precision is different as the variable precision.


- as example if the default precision is 64 bit and the variable is 64 bit I call both (python check and numpy check) because it can be anyone of them
- in the other hand the is no need to call python check

Reply:
so i had 2 choice either change argument to np.float32 or change decorator to real

Reply:
Ah ok no problem then :+1:
Although would you mind using `'float'` instead of `'real'`? `real` is the name of floating point types in fortran, but not in any other language. Everywhere else `real` is the real part of a complex number. `PythonReal` and `NumpyReal` have caused confusion a few times because we insist on using `real` everywhere in pyccel for `float`

Reply:
'real' is used in a lot of place do you want me to change it everywhere ?

Reply:
No. It's fine. I just don't want to add to the problem. If you could just change it in the 3/4 places you have added it that would be great

Reply:
done in ceef690